src/runtime-rs/crates/hypervisor/src/kernel_param.rs

```diff
@@ -60,10 +55,6 @@ impl KernelParams {
             Param::new("systemd.mask", "systemd-networkd.service"),
         ];
 
-        if debug {
-            params.push(Param::new(LOG_VPORT_OPTION, VSOCK_LOGS_PORT));
```

Comment:
I think it's better to export the guest kernel log here. This function is used as the util function for the kernel parameters.

Reply:
Yes, as I have thought that `agent.log_vport` will not used for all hypervisor/agent, so I move this branch logic to dragonball's implementation to make KernelParams works as a more independent module.

If we still want to export guest kernel log here, we should at least determine which vmm is currently in use. And as KernelParam can not fetch the config, we may need to add a parameter to this function. Thus it would be like:
in KernelParam:
```
pub(crate) fn new(debug: bool, hypervisor: &str) -> self {
	...
	if debug {
		if hypervisor == "dragonball" {
			...
		}
	}
}
```

in DragonballInner:
```
	...
	KernelParam::new(config.enable_debug, "dragonball");
	...
```

Reply:
ye, I see, I just check the `go runtime`, `agent.log_vport` will only work for `firecracker`.